cves/CVE-2015-8364.yml

```diff
@@ -64,7 +64,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Heap-based interger overflow vulnerabilty that allowed remote attackers to 
+  cause a denial-of-service attack or other unintended behavior by specifying too large image file 
+  dimensions in an interactive session of Indeo, a compression / decompression technology.
```

Comment:
This is good but I could use a bit more about what the unintended behavior - especially what heap-based means for it and how that applies to the integer. Or think of it this way: what is that integer used for that leads to bad behavior?